pkg/policy/condition/key.go

```diff
@@ -76,6 +76,15 @@ const (
 
 	// AWSEpochTime - key representing the current epoch time.
 	AWSEpochTime = "aws:EpochTime"
+
+	// AWSPrincipalType - user principal type currently supported values are "User" and "Anonymous"
+	AWSPrincipalType = "aws:principaltype"
+
+	// AWSUserID - user unique ID, in Minio this value is same as username
+	AWSUserID = "aws:userid"
+
+	// AWSUsername - user friendly name, in Minio this value is same as userid
```

Comment:
same as username?

Reply:
it was to mention that they both carry same values

Reply:
The comment can be better i.e. `// AWSUserID - user unique ID.  In Minio this is access key`